opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -103,6 +103,11 @@ where
         log_record.severity_number = Some(map_severity_to_otel_severity(meta.level().as_str()));
         log_record.severity_text = Some(meta.level().to_string().into());
 
+        // add the `name` metadata to attributes
+        // TBD - Propose this to be part of log_record metadata.
+        let vec = vec![("name", meta.name())];
```

Comment:
just to confirm: OTLPExporters will export this as "name" ?

Reply:
yes, this will go within attributes across all the log exporters.